pkg/console/install_panels.go

```diff
@@ -1162,28 +1170,72 @@ func addNetworkPanel(c *Console) error {
 		mgmtNetwork.Gateway = gateway
 		return "", nil
 	}
-	gatewayVConfirm := func(g *gocui.Gui, v *gocui.View) error {
-		msg, err := validateGateway()
+	gatewayVConfirm := gotoNextPanel(c, []string{mtuPanel}, validateGateway)
+	gatewayV.KeyBindings = map[gocui.Key]func(*gocui.Gui, *gocui.View) error{
+		gocui.KeyArrowUp: gotoNextPanel(c, []string{addressPanel}, func() (string, error) {
+			mgmtNetwork.Gateway, err = gatewayV.GetData()
+			return "", err
+		}),
+		gocui.KeyArrowDown: gatewayVConfirm,
+		gocui.KeyEnter:     gatewayVConfirm,
+		gocui.KeyEsc:       gotoPrevPage,
+	}
+	setLocation(gatewayV.Panel, 3)
+	c.AddElement(gatewayPanel, gatewayV)
+
+	// mtuV
+	mtuV.PreShow = func() error {
+		c.Gui.Cursor = true
+		if mgmtNetwork.MTU == 0 {
```

Comment:
I'd suggest if the user doesn't specify any value, just leave it empty in the wicked configuration file and let wicked handle it.

Reply:
Updated it, thanks!